modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/HeapLockManager.java

```diff
@@ -80,7 +78,12 @@
  * <p>Additionally limits the lock map size.
  */
 public class HeapLockManager extends AbstractEventProducer<LockEvent, LockEventParameters> implements LockManager {
-    private static final IgniteLogger LOG = Loggers.forClass(HeapLockManager.class);
+    /** Throttled logger. */
+    private static final IgniteLogger THROTTLED_LOG = Loggers.toThrottledLogger(
+            Loggers.forClass(HeapLockManager.class),
+            // TODO: IGNITE-24181 Get rid of Common thread pool.
+            ForkJoinPool.commonPool()
```

Comment:
Why `ForkJoinPool` there? May we pass `ThreadPoolsManager#commonScheduler` there from `IgniteImpl` in time? I saw `IGNITE-24181`, but just thought that common scheduler may be more reasonable for now.

Reply:
I do not want to drag an executor here because it would be deleted in the mentioned ticket.

Reply:
Up to you, accepted